### PR TITLE
Order service need options

### PR DESCRIPTION
**Pull request by apeltola**

### CODE REVIEW

File: service/src/main/resources/db/migration/V99__service_need_option_display_order.sql [hunk 1]
```diff
@@ -0,0 +1,2 @@
+ALTER TABLE service_need_option
+    ADD COLUMN display_order integer DEFAULT NULL;
```
Comment (kimifoud) [inline service/src/main/resources/db/migration/V99__service_need_option_display_order.sql:2]: newline
```suggestion
    ADD COLUMN display_order integer DEFAULT NULL;

```

File: service/src/main/kotlin/fi/espoo/evaka/serviceneednew/NewServiceNeedQueries.kt [hunk 2]
```diff
@@ -250,6 +250,7 @@ fun Database.Read.getServiceNeedOptionPublicInfos(placementTypes: List<Placement
             valid_placement_type
         FROM service_need_option
         WHERE default_option IS FALSE AND valid_placement_type = ANY(:placementTypes::placement_type[])
+        ORDER BY display_order;
```
Comment (kimifoud) [inline service/src/main/kotlin/fi/espoo/evaka/serviceneednew/NewServiceNeedQueries.kt:253]:
```suggestion
        ORDER BY display_order
```

**Review by kimifoud — APPROVED** [review]

**Review by kimifoud — CHANGES_REQUESTED** [review]
`migrations.txt` needs to be updated

**Review by kimifoud — APPROVED** [review]

**Comment by kimifoud** [comment]
> Conflicting files
> `service/src/main/resources/migrations.txt`

🥳

File: service/src/main/resources/migrations.txt [hunk 3]
```diff
@@ -98,4 +98,5 @@ V97__assistance_action_option.sql
 V98__add_type_to_daily_service_times.sql
 V99__change_sleepinghours_to_sleepingminutes.sql
 V100__child_images.sql
-V101__update_fee_decision_pricing.sql
+V101__service_need_option_display_order.sql
+V102__update_fee_decision_pricing.sql
```
Comment (kimifoud) [inline service/src/main/resources/migrations.txt:102]: 🤔

Comment (apeltola) [reply service/src/main/resources/migrations.txt:102]: 🤦🏻

**Pull request merged**